src/App.jsx

```diff
@@ -1,14 +1,23 @@
 /* eslint-disable prettier/prettier */
+import { BrowserRouter as Router, Switch, Route, NavLink } from 'react-router-dom';
 import BookList from './views/Books/BookList'
 import BookDetail from './views/Books/BookDetail'
 import './App.css'
+import Home from './components/home/Home';
+
 
 function App() {
   // TODO: Add routes to books & views
   return (
     <main className="container">
       <h1>Library Catalog</h1>
-      <BookList />
+      <Router>
+        <Switch>
+          <Route exact path ='/' component={Home} />
+          <Route path='/books/:id' component={BookDetail} />
```

Comment:
Be careful putting the `/:id` route above the base route here. Anything after the slash will be interpreted as an "id" and can cause unexpected behaviors. If you structure the routes this way, you want to be sure to use the `exact` keyword, otherwise the `/:id` route should go beneath the base route.